cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -68,7 +68,7 @@ import org.ossreviewtoolkit.utils.storage.FileArchiver
 
 class ReporterCommand : CliktCommand(
     name = "report",
-    help = "Present Analyzer and Scanner results in various formats."
+    help = "Present Analyzer, Scanner and Evaluator results in various formats."
```

Comment:
out-of-scope: This sentence is a bit mis-leading IMO in the sense that it suggests that the reporter input is only the ORT result, while in fact it also consumes resolutions, package configurations and so forth which are not part of the ORT result.

Reply:
I think it's still good enough for a short description, at least it's more correct than before.